Makefile

```diff
@@ -1,14 +1,29 @@
-db:
+db_postgres:
 	docker run --rm -d --name postgres -p 5432:5432 \
   -e POSTGRES_DB=fang \
   -e POSTGRES_USER=postgres \
   -e POSTGRES_PASSWORD=postgres \
   postgres:latest
+
 clippy:
-	cargo clippy --all-features
-diesel:
-	cd fang && DATABASE_URL=postgres://postgres:postgres@localhost/fang diesel migration run
-stop:
+	cargo clippy --verbose --all-targets --all-features -- -D warnings
+diesel_postgres:
+	cd fang/postgres_migrations && DATABASE_URL=postgres://postgres:postgres@localhost/fang diesel migration run
+
+diesel_mysql:
+	cd fang/mysql_migrations && DATABASE_URL=mysql://root:fang@127.0.0.1/fang diesel setup
```

Comment:
Shouldn't this be `diesel migration run` like in the Postgres case?

Reply:
yup fixed here

[sqlite migrations](https://github.com/ayrat555/fang/pull/134/commits/6a6f5547129037fe46c0b440262acf1081845990)